src/AspNet.Security.OAuth.SuperOffice/SuperOfficeAuthenticationHandler.cs

```diff
@@ -112,6 +89,54 @@ public class SuperOfficeAuthenticationHandler : OAuthHandler<SuperOfficeAuthenti
             return new AuthenticationTicket(context.Principal, context.Properties, Scheme.Name);
         }
 
+        private async Task<string> ProcessIdTokenAndGetContactIdentifier(
```

Comment:
Style nit: add `Async` suffix to method name please.

Reply:
Of course. Thanks.